### PR TITLE
remove ::shadow references

### PR DESCRIPTION
**Pull request by notwaldorf**
:dragon_face:

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -73,14 +73,9 @@
...
-        width: 100px;
-        height: 100px;
-      }
-
-      paper-icon-button.huge #icon {
-        width: 100px;
-        height: 100px;
+      paper-icon-button.huge {
+        --iron-icon-height: 100px;
+        --iron-icon-width: 100px;
```
Comment (cdata) [inline demo/index.html]: Does this actually work? I thought that this had to be assigned in a `:host` or `:root`..

Comment (addyosmani) [inline demo/index.html]: Working for me inside `custom-style` as per above http://jsbin.com/pojatemobe/edit?html,output but afaik it should also work inside `:root`.

Comment (notwaldorf) [inline demo/index.html]: I think that it should work. Or rather; that's how we (I) have been using it for a long time :)

**Comment by notwaldorf** [comment]
@cdata Fixeded!

File: demo/index.html [hunk 2]
```diff
@@ -70,17 +70,8 @@
...
-
-      paper-icon-button.huge::shadow #icon {
-        width: 100px;
-        height: 100px;
-      }
-
-      paper-icon-button.huge #icon {
-        width: 100px;
         height: 100px;
+        --paper-icon-button-ink-color: var(--paper-indigo-500);
```
Comment (cdata) [inline demo/index.html]: :purple_heart:

File: paper-icon-button.html [hunk 3]
```diff
@@ -99,6 +92,11 @@
         cursor: auto;
         @apply(--paper-icon-button-disabled);
       }
+
+      iron-icon {
+        --iron-icon-width: 100%;
+        --iron-icon-height: 100%;
```
Comment (cdata) [inline paper-icon-button.html]: :+1:

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**